src/service/auth/service.ts

```diff
@@ -0,0 +1,14 @@
+import http from '../http.service';
+import type { LogInType } from './types';
+
+const UserService = {
+  login: async (data: LogInType) => {
+    const response = await http.POST(
+      'http://24.199.108.242/v1/auth/login',
```

Comment:
The http.service file already includes the API_URL, you'll only need to pass the path:` /auth/login`. Also, when you need to use URLs please always use the .env file in the root of the application.